docs/SMCWG-charter.md

```diff
@@ -162,8 +162,8 @@ forth in the Bylaws.
    * The organization name, as they wish it to appear on the Forum Web site and
      in official Forum documents.
    * URL of the applicant’s main Web site.
-   * Names and email addresses of employees who will participate in the Working
-     Group and Forum as Member representatives.
+   * Names and email addresses of designated representatives who will participate 
+     in the Working Group and Forum on behalf of the Member.
```

Comment:
From https://github.com/sleevi/cabforum-docs/pull/9/files/7d20e6bcfb580b409a37b8f2f1d0b43d6d10e749:

designated representatives need not be employees.